JavaScript/JSON/Types/Internal.hs

```diff
@@ -324,5 +323,5 @@ foreign import javascript unsafe
   js_listAssocs :: SomeObject m -> Exts.State# s -> (# Exts.State# s, Exts.Any {- [(JSString, Value)] -} #)
 
 foreign import javascript unsafe
-  "JSON.stringify($1)"
+  "JSON.stringify"
```

Comment:
Our current codegen means that the arguments are essentially appended on as a string, so that `JSON.stringify` becomes `JSON.stringify(args)` at use sites. It would be possible to use the fat arrow syntax, but redundant.